### DISCUSSION TITLE
Tanix W2 no link

### DISCUSSION THREAD

**Discussion by mikhail032** [devmfc/debian-on-amlogic#74] [Q&A]
Tanix W2. The wired network does not work, but everything works on Android.
![photo_2024-07-15_11-35-36](https://github.com/user-attachments/assets/3d84b502-e850-44b7-b1e0-6ce635f217df)

**Comment by ghost** [comment]
try to use generic905w2 dtb instead of tanixw2 dtb...

**Reply by mikhail032** [reply]
Thanks, it worked!
